plugins/net_plugin/tests/auto_bp_peering_unittest.cpp

```diff
@@ -38,12 +38,12 @@ struct mock_connections_manager {
 
 struct mock_net_plugin : eosio::auto_bp_peering::bp_connection_manager<mock_net_plugin, mock_connection> {
 
-   bool                         is_in_sync = false;
+   bool                         is_not_lib_catchup = false;
    mock_connections_manager     connections;
    std::vector<std::string>     p2p_addresses{"0.0.0.0:9876"};
    const std::string&           get_first_p2p_address() const { return *p2p_addresses.begin(); }
 
-   bool in_sync() { return is_in_sync; }
+   bool not_lib_catchup() { return is_not_lib_catchup; }
```

Comment:
I don't like these negative booleans personally, for example this is harder to parse for me:
`plugin.is_not_lib_catchup = true;`

than:

`plugin._lib_catchup = false;`

I think it would be better to have:
`bool lib_catchup() { return _lib_catchup; }`

Reply:
renamed